src/WebJobs.Script/Config/TransmissionStatusTelemetryModule.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.ApplicationInsights.Extensibility.Implementation;
+using Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    /// <summary>
+    /// Transmission Status Telemetry Module
+    /// </summary>
+    internal class TransmissionStatusTelemetryModule : ITelemetryModule, IDisposable
+    {
+        private bool isInitialized = false;
+        private DiagnosticListener source = new DiagnosticListener(ScriptConstants.HostDiagnosticSourcePrefix + "ApplicationInsights");
```

Comment:
Use the `_`  prefix to be consistent with the rest of the code base.

```suggestion
        private readonly DiagnosticListener _source
```

Reply:
Same comment for other private variables.